src/test/java/org/jabref/logic/integrity/PersonNamesCheckerTest.java

```diff
@@ -62,4 +69,29 @@ public void validCorporateNameAndPerson() throws Exception {
         assertEquals(Optional.empty(), checker.checkValue("{JabRef} and Stefan Kolb"));
         assertEquals(Optional.empty(), checker.checkValue("{JabRef} and Kolb, Stefan"));
     }
+
+    @ParameterizedTest
+    @MethodSource("provideCorrectFormats")
+    public void authorNameInCorrectFormatsShouldNotComplain(String input) {
+        assertEquals(Optional.empty(), checkerb.checkValue(input));
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideIncorrectFormats")
+    public void authorNameInIncorrectFormatsShouldComplain(String input) {
+        assertNotEquals(Optional.empty(), checkerb.checkValue(input));
+    }
+
+    private static String[] provideCorrectFormats() {
+        String[] data = new String[] {"", "Knuth", "Donald E. Knuth and Kurt Cobain and A. Einstein"};
```

Comment:
Since a single `String` string is required at each test, please use following code:

```suggestion
        return Stream.of("", "Knuth", "Donald E. Knuth and Kurt Cobain and A. Einstein");
```

This is more readble than `String[]`. The return type will be `Stream<String>`.